language.go

```diff
@@ -10,11 +10,16 @@ var (
 )
 
 func init() {
-	languageGeneral = regexp.MustCompile(`(?i)\b(?:RUS|NL|FLEMISH|GERMAN|DUBBED|FR(?:ENCH)?|Truefrench|VF(?:[FI])|VOST(?:(?:F(?:R)?)|A)?|SUBFRENCH|MULTi(?:Lang|-VF2)?|ITA(?:LIAN)?|(?:iTALiAN))\b`)
+	languageGeneral = regexp.MustCompile(`(?i)\b(?:RUS|NL|FLEMISH|GERMAN|DUBBED|FR(?:ENCH)?|Truefrench|VF(?:[FI])|VOST(?:(?:F(?:R)?)|A)?|SUBFRENCH|MULTi(?:Lang|-VF2)?|ITA(?:LIAN)?|(?:iTALiAN)|Eng)\b`)
 }
 
-func (p *parser) GetLanguage() string {
-	return p.FindString("language", languageGeneral, FindStringOptions{Handler: func(str string) string {
-		return strings.ToLower(str)
+func (p *parser) GetLanguage() []string {
```

Comment:
Change to `GetLanguages` since it returns a slice now.

Reply:
Done!